pages/pricing/_benefits.html

```diff
@@ -3,8 +3,8 @@
   <li><a href="/training/" class="text-white">DevOps Training Library</a>: A collection of training video courses that teach you DevOps.</li>
   <li><a href="/houston/" class="text-white">Gruntwork Houston</a>: Mission Control for your infrastructure. Currently in private beta.</li>
   {% if include.dedicated_support %}
-    <li><a href="/support/" target="_blank" class="text-white">Dedicated Support</a>: Get support via email, chat, and phone/video from Gruntwork. 1 Business Day SLA.</li>
+    <li><a href="/support/" target="_blank" class="text-white">Dedicated Support</a>: Get support via email, chat, and phone/video from Gruntwork. 2 Business Day SLA.</li>
   {% else %}
-    <li><a href="https://community.gruntwork.io/" target="_blank" class="text-white">Community Forum</a>: Get support via our public community forum. 2 Business Day SLA.</li>
+    <li><a href="https://community.gruntwork.io/" target="_blank" class="text-white">Community Support</a>: Chat with fellow customers and  occasionally Gruntwork engineers via Slack.</li>
```

Comment:
This is still linking to the community forum. Should link to gruntwork.io/support

Reply:
Good catch. Fixed in 8eb59e4033829db0a7df7c1b59c3b690349c0b5f.